src/Nox.Ui.Blazor.Lib/Components/Generic/SelectList.razor

```diff
@@ -0,0 +1,26 @@
+﻿@namespace Nox.Ui.Blazor.Lib.Components.Generic
+@using MudBlazor
+@using Nox.Ui.Blazor.Lib.Models
+
+@if (SelectionList.Any())
+{
+    <MudSelect Label="@Title"
+               Variant="Variant.Outlined"
+               MultiSelection="true"
+               FullWidth="true"
+               Clearable="true"
+               Required="@Required"
+               RequiredError="@ErrorRequiredMessage(Title)"
+               TextChanged="OnIdChanged"
+               Value="@CurrentIdList"
+               Text="@DisplaySelectedList">
+        @foreach (SelectEntityModel item in SelectionList)
+        {
+            if (!string.IsNullOrWhiteSpace(item.Id)
```

Comment:
@DanielHodgeGH  this safe guard can hide or is hidding some issue in the domain

can not exist an entity without an Id or a text field, we should not support such cases

I would prefer to remove the safe guard and see if we need to fix anything in other place?